public/app/core/components/ForgottenPassword/ForgottenPassword.tsx

```diff
@@ -32,7 +33,7 @@ export const ForgottenPassword: FC = () => {
       <div>
         <p>An email with a reset link has been sent to the email address. You should receive it shortly.</p>
         <Container margin="md" />
-        <LinkButton variant="primary" href="/login">
+        <LinkButton variant="primary" href={getConfig().appSubUrl + '/login'}>
```

Comment:
May I suggest creating/using a variable `loginHref` with the value `getConfig().appSubUrl + '/login'` so future changes can be done in one place, wdyt @KamalGalrani ?